symplyphysics/laws/hydro/hydrostatic_pressure_from_density_and_depth_acceleration.py

```diff
@@ -0,0 +1,28 @@
+from sympy import Eq, solve
+from symplyphysics import units, Quantity, Symbol, print_expression, validate_input, validate_output
+
+# Description
+# Law: P = ρ * g * h
+# P - hydrostatic pressure
+# ρ - density of liquid
+# g - acceleration of gravity
```

Comment:
We already have this law. Use any acceleration here and explain how it should be directed, to make this law valid.